core/tests/protractor_desktop/topicAndStoryEditorFileUploadFeatures.js

```diff
@@ -185,7 +185,13 @@ describe('Chapter editor functionality', function() {
     for (var i = 0; i < numExplorations; i++) {
       var info = dummyExplorationInfo.slice();
       info[0] += i.toString();
-      await workflow.createAndPublishExploration.apply(workflow, info);
+      if (i === 0) {
+        info.push(true);
+        await workflow.createAndPublishExploration.apply(workflow, info);
+      } else {
+        info.push(false);
+        await workflow.createAndPublishExploration.apply(workflow, info);
+      }
```

Comment:
What about this?

```suggestion
      await workflow.createAndPublishExploration.apply(workflow, i === 0);
```

Reply:
@kevintab95 As the second parameter is the list (info). I have updated the above :

info.push(i === 0);
await workflow.createAndPublishExploration.apply(workflow,info);

Reply:
Yep sounds good!